kubejs/server_scripts/enigmatica/kubejs/functions.js

```diff
@@ -76,3 +76,11 @@ const playerHas = (item, player) => {
 function lowerTiers(tiers, tier) {
     return tiers.slice(0, tiers.indexOf(tier));
 }
+
+// transplant the md5 from `<type's mod>:kjs_<md5>` onto the supplied prefix
+function md5(recipe, id_prefix) {
+    if (recipe.getId().includes(':kjs_')) {
+        recipe.serializeJson(); // without this the hashes *will* collide
```

Comment:
got any idea what the performance impact of this is?

Reply:
Alright 👍 I'd really like a few before-after measurements of this before merging :D

Reply:
It seems to vary wildly, several expert/normal mode runs linked above have higher/lower values when you search for `Added recipes in` in their logs, but it seems to average out to 2.5 seconds each, if this pull makes any impact it appears to be less than half a second 🤔 

Ideally i would pull all their logs programmatically and put the values in a table, but currently the different seems negligible compared to the e6 world/server load times we're accustomed to ^-^